From c646b900c72cd85b8476b49249b29f955ab07a09 Mon Sep 17 00:00:00 2001
From: Stephan Bergmann <sbergman@redhat.com>
Date: Thu, 4 Oct 2012 22:46:10 +0200
Subject: [PATCH] fdo#52022: Simple/LargeControlPoints actually can have nil
 values

...in old LO 3.5 registrymodifications.xcu, which causes trouble when migrating
to >= LO 3.6, see <https://bugs.freedesktop.org/show_bug.cgi?id=52022#c23> for
all the details.

Change-Id: I8fa1e67998c9588b9d8bf3a3b52e4b0684d3d705
(cherry picked from commit a51833bf79eed21a7cddd6da45dc9d3dc07b4983)
---
 officecfg/registry/schema/org/openoffice/Office/Calc.xcs      | 4 ++--
 officecfg/registry/schema/org/openoffice/Office/Writer.xcs    | 4 ++--
 officecfg/registry/schema/org/openoffice/Office/WriterWeb.xcs | 4 ++--
 3 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/officecfg/registry/schema/org/openoffice/Office/Calc.xcs b/officecfg/registry/schema/org/openoffice/Office/Calc.xcs
index 0f5898e..55ee57a 100644
--- a/officecfg/registry/schema/org/openoffice/Office/Calc.xcs
+++ b/officecfg/registry/schema/org/openoffice/Office/Calc.xcs
@@ -340,7 +340,7 @@
 					</info>
 					<value>false</value>
 				</prop>
-				<prop oor:name="SimpleControlPoint" oor:type="xs:boolean" oor:nillable="false">
+				<prop oor:name="SimpleControlPoint" oor:type="xs:boolean">
 					<!-- OldPath: Calc/Layout/Lines -->
 					<!-- OldLocation: Soffice.cfg -->
 					<!-- UIHints: Tools - Options -Spreadsheets - Layout - [Section] Lines -->
@@ -351,7 +351,7 @@
 					</info>
 					<value>false</value>
 				</prop>
-				<prop oor:name="LargeControlPoint" oor:type="xs:boolean" oor:nillable="false">
+				<prop oor:name="LargeControlPoint" oor:type="xs:boolean">
 					<!-- OldPath: Calc/Layout/Lines -->
 					<!-- OldLocation: Soffice.cfg -->
 					<!-- UIHints: Tools - Options -Spreadsheets - Layout - [Section] Lines -->
diff --git a/officecfg/registry/schema/org/openoffice/Office/Writer.xcs b/officecfg/registry/schema/org/openoffice/Office/Writer.xcs
index e797988..8865f74 100644
--- a/officecfg/registry/schema/org/openoffice/Office/Writer.xcs
+++ b/officecfg/registry/schema/org/openoffice/Office/Writer.xcs
@@ -1306,7 +1306,7 @@
 					</info>
 					<value>false</value>
 				</prop>
-				<prop oor:name="SimpleControlPoint" oor:type="xs:boolean" oor:nillable="false">
+				<prop oor:name="SimpleControlPoint" oor:type="xs:boolean">
 					<!-- OldPath: Writer/Layout/Lines -->
 					<!-- OldLocation: Soffice.cfg -->
 					<!-- UIHints: Tools - Options - Text document - Layout - [Section] Lines -->
@@ -1317,7 +1317,7 @@
 					</info>
 					<value>false</value>
 				</prop>
-				<prop oor:name="LargeControlPoint" oor:type="xs:boolean" oor:nillable="false">
+				<prop oor:name="LargeControlPoint" oor:type="xs:boolean">
 					<!-- OldPath: Writer/Layout/Lines -->
 					<!-- OldLocation: Soffice.cfg -->
 					<!-- UIHints: Tools - Options - Text document - Layout - [Section] Lines -->
diff --git a/officecfg/registry/schema/org/openoffice/Office/WriterWeb.xcs b/officecfg/registry/schema/org/openoffice/Office/WriterWeb.xcs
index 79f581a..6b99869 100644
--- a/officecfg/registry/schema/org/openoffice/Office/WriterWeb.xcs
+++ b/officecfg/registry/schema/org/openoffice/Office/WriterWeb.xcs
@@ -210,7 +210,7 @@
 					</info>
 					<value>false</value>
 				</prop>
-				<prop oor:name="SimpleControlPoint" oor:type="xs:boolean" oor:nillable="false">
+				<prop oor:name="SimpleControlPoint" oor:type="xs:boolean">
 					<!-- OldPath: HTML_Editor/Layout/Lines -->
 					<!-- OldLocation: Soffice.cfg -->
 					<!-- UIHints: Tools - Option - HTML-Document - Layout - [Section] Lines -->
@@ -221,7 +221,7 @@
 					</info>
 					<value>false</value>
 				</prop>
-				<prop oor:name="LargeControlPoint" oor:type="xs:boolean" oor:nillable="false">
+				<prop oor:name="LargeControlPoint" oor:type="xs:boolean">
 					<!-- OldPath: HTML_Editor/Layout/Lines -->
 					<!-- OldLocation: Soffice.cfg -->
 					<!-- UIHints: Tools - Option - HTML-Document - Layout - [Section] Lines -->
-- 
1.7.11.4

